spec/api/v1/posters_controller_spec.rb

```diff
@@ -156,15 +161,16 @@
     # puts poster
     deleted_poster = Poster.find_by(id: id)
 
-    expect(response).to have_http_status(204)
+    expect(response).to have_http_status(200)
```

Comment:
I think we do need to keep it at 204 per the assignment: "NOT return any JSON body at all, and should return a 204` HTTP status code"
The problem is that the code is currently returning a "1" in the JSON body and it should be empty, so we still have to debug this.